### PR TITLE
Prepare release 0.6.4.1

### PR DESCRIPTION
**Pull request by sjakobi**

### CODE REVIEW

File: containers/changelog.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog for [`containers` package](http://github.com/haskell/containers)
 
+## [0.6.4.1]
+
+### Bug fixes
+
+* [Replace value-forcing variants of `compose` with lazy variants.](https://github.com/haskell/containers/pull/745)
```
Comment (sjakobi) [inline containers/changelog.md:7]: Any ideas how to improve this wording?!

**Review by sjakobi — COMMENTED** [review]
It would be nice to make a Hackage candidate, so we can check that everything's fine with the metadata. I don't have the necessary permissions though.

File: containers/changelog.md (hunk 1 above)
Comment (treeowl) [reply containers/changelog.md:7]: Nope.

**Review by treeowl — APPROVED** [review]
LGTM

**Comment by treeowl** [comment]
Once you've merged this (I dunno if you want to preserve the commits or not) I'll put up a Hackage package candidate. If Hackage accepts it, I'll tag the release.

**Comment by sjakobi** [comment]
Cheers, @treeowl! This is ready to go.

**Comment by sjakobi** [comment]
@treeowl What's the status here?

**Pull request merged**